sample_queries/README.md

```diff
@@ -0,0 +1,13 @@
+# Sample Queries
+
+This folder contains sample queries showing the functionality of querying a
+dataset in BigQuery which was imported with Variant Transforms.
+
```

Comment:
How about adding this paragraph? Please feel free to modify it as you see suited.

By providing these queries we are hoping researchers can better utilize their BigQuery tables. Also this will provide a way for the community to contribute by modifying these queries or suggesting new ones. We highly encourage all users to send us their queries so we can share them here with all other researchers. Please feel free to [submit an issue](https://github.com/googlegenomics/gcp-variant-transforms/issues) or contact us via our public mailing list <gcp-life-sciences-discuss@googlegroups.com>.